src/views/RecIM/components/List/Listing/index.jsx

```diff
@@ -351,13 +354,23 @@ const TeamListing = ({ team, invite, match, setTargetTeamID, callbackFunction })
   );
 };
 
-// We could also use ParticipantID (not student ID) if we have that and prefer it to AD_Username
-const ParticipantListing = ({ participant, minimal, callbackFunction, showParticipantOptions }) => {
-  const { teamID } = useParams();
+const ParticipantListing = ({
+  participant,
+  minimal,
+  callbackFunction,
+  showParticipantOptions,
+  withAttendance,
+  isAdmin,
+  initialAttendance,
+  teamID,
+  matchID,
+}) => {
+  const { teamIDParam } = useParams(); // @TODO will fail if not on team page
```

Comment:
If you're passing teamID in, why not just have a conditional like
`_teamID = teamID ?? teamIDParam `
(not sure which way the logic goes)